pkgs/love2d/example/assets/pocket_bomber/main.lua

```
@@ -0,0 +1,103 @@
+end
+
+function love.keypressed(key)
+    Input.keypressed(key)
+    StateMachine.keypressed(key)
+end
+
+function love.keyreleased(key)
+    StateMachine.keyreleased(key)
+end
+
+function love.touchpressed(id, x, y, dx, dy, pressure)
```

Luacheck (1.2.0)

[warning] unused argument 'dx'

(W212)